samples/artifacts/DataSummarization/NamedWindow.siddhi

```diff
@@ -0,0 +1,52 @@
+@App:name("NamedWindow")
+@App:description("Demonstrates how to create a named window for better query performance.")
```

Comment:
Review descriptions and comments -> Named window another way to define windows so that it can be shared across various queries

Reply:
6abc5699d9a9c3ddf8f5e200716a4e7c236ef463 This commit fixed the above correction